docs/building-modules/querier.md

```diff
@@ -15,18 +15,39 @@ A `Querier` designates a function that processes [`queries`](./messages-and-quer
 
 The `querier` type defined in the Cosmos SDK specifies the typical structure of a `querier` function:
 
-+++ https://github.com/cosmos/cosmos-sdk/blob/7d7821b9af132b0f6131640195326aa02b6751db/types/queryable.go#L6
++++ https://github.com/cosmos/cosmos-sdk/blob/9a183ffbcc0163c8deb71c7fd5f8089a83e58f05/types/queryable.go#L9
 
 Let us break it down:
 
 - The `path` is an array of `string`s that contains the type of the query, and that can also contain `query` arguments. See [`queries`](./messages-and-queries.md#queries) for more information.
-- The `req` itself is primarily used to retrieve arguments if they are too large to fit in the `path`. This is done using the `Data` field of `req`. 
-- The [`Context`](../core/context.md) contains all the necessary information needed to process the `query`, as well as a cache-wrapped copy of the latest state. It is primarily used by the [`keeper`](./keeper.md) to access the state. 
-- The result `res` returned to `baseapp`, marhsalled using the application's [`codec`](../core/encoding.md). 
+- The `req` itself is primarily used to retrieve arguments if they are too large to fit in the `path`. This is done using the `Data` field of `req`.
+- The [`Context`](../core/context.md) contains all the necessary information needed to process the `query`, as well as a cache-wrapped copy of the latest state. It is primarily used by the [`keeper`](./keeper.md) to access the state.
+- The result `res` returned to `baseapp`, marshalled using the application's [`codec`](../core/encoding.md).
 
 ## Implementation of a module `querier`s
```

Comment:
not sure if we want to mix the term Querier with gRPC as they represent different implementations. Probably
```suggestion
## Implementation of a module query service
```